### ISSUE TITLE
1 CSS property is overridden

### ISSUE THREAD

**Issue by Geekcoder555** [systers/systers.github.io#144]
## Description
1 CSS property is overridden

Generally, it is unnecessary to define 2 times the same property within the same rule: the second one overrides the first.
This may affect the readability of CSS code. Eliminate unnecessarily overridden properties also reduce the file size.

How to improve it?
Remove one occurrence of the duplicated property. For example, the following properties:

## Mocks
[INSERT RELEVANT PNG FILE]
.myClass {
  margin: 20px;
  ...
  margin: 10px;
}
Should be replaced by:

.myClass {
  margin: 10px;
}
main-menu:target + .bac[...]ckdrop {background: rgba(0,0,0,.85)} (line 129, col 3)
## Acceptance Criteria
styles.css should be modified accordingly.
### Update [Required]
- [ ] styles.css

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
5-10 min

**Comment by akshitac8** [comment]
Hey,
Is this bug open for solving?

**Comment by sara-02** [comment]
@AnkitKhatri55555 Can you please let @akshitac8 know if this issue is being worked upon or not?

**Comment by AnkitKhatri55555** [comment]
Ya, @akshitac8 ,u can go ahead

**Comment by divyanshu-rawat** [comment]
yup @akshitac8 Go Ahead 👍

**Comment by akshitac8** [comment]
Hey @AnkitKhatri55555 I have cleaned the CSS and deployed a [demo](https://systers.herokuapp.com/) website. It would be great if you can have a look. Also, the changes made does not change any kind of appearance for the [Systers](http://systers.io/) website. Should I generate a PR?

**Comment by AnkitKhatri55555** [comment]
Ya sure generate a PR, but some time ago ,we received a PR regarding the same

**Comment by akshitac8** [comment]
There seems to be some error in PR #146 given by Codacy. I think there were multiple changes done by @Vertika-Shukla That error is not shown in PR #147 as I have only changed only the required lines.

**Comment by akshitac8** [comment]
I have also tried to use the least number of lines in *styles.css* These changes are still not committed in the repo but reflect on [Heroku](https://systers.herokuapp.com/). If those look fine @AnkitKhatri55555 may I commit those as well in a separate PR.

**Comment by AnkitKhatri55555** [comment]
@akshitac8 , I think that's the error only for the case regarding usage of important as it's not encouraged to use

**Comment by AnkitKhatri55555** [comment]
Don't create a seperate PR

**Comment by akshitac8** [comment]
I agree @AnkitKhatri55555 The PR #146 shows the following lines were added in *styles.css* but I don't think they were required as far as this issue is concerned.
```
styles.css
+
+  .main-menu a {
+    color: rgb(98,181,229);
+    border: 0 !important; /* Remove borders from off-canvas styling */
```

**Comment by AnkitKhatri55555** [comment]
Yes,@akshitac8 I agree, but I think in PR #146 , the changes were made at the required lines but she cut paste the code in styles.css ,so the addition is reflected

**Comment by divyanshu-rawat** [comment]
Ya, @akshitac8 don't create a separate PR 👍.
And Yes, @AnkitKhatri55555 she just copy pasted the code that's why it is reflecting +491 changes with 0 deletions.

**Comment by akshitac8** [comment]
@divyanshu-rawat I have already created PR #147 as I asked @AnkitKhatri55555 before. Is that PR ready to be merged or should I make any other changes?

**Comment by divyanshu-rawat** [comment]
@akshitac8 I have replied to your Query in the PR 👍

**Comment by sara-02** [comment]
@divyanshu-rawat #146 is also there, but @Vertika-Shukla you never asked for assignment of the issue. Please take of this, do not start working on the issue before making sure it is not assigned to someone else, helps avoid confusion for everyone :)

**Comment by Vertika-Shukla** [comment]
@sara-02 Sure, I will follow this from next time :)
